app/ssttpcalculator/BigDecimalUtil.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ssttpcalculator
+
+object BigDecimalUtil {
+
+  def roundDownToNearestTen(value: BigDecimal): BigDecimal = BigDecimal((value.longValue() / 10) * 10)
```

Comment:
Nb..
The BigDecimal.longValue can be a truncating operation - the docs at https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#longValue-- state "any fractional part of this BigDecimal will be discarded"